### ISSUE TITLE
Custom step layout issue

### ISSUE THREAD

**Issue by Resource-4** [alantoa/react-native-awesome-slider#79]
Expected OUTPUT :- 
<img width="474" alt="image" src="https://github.com/user-attachments/assets/56cf8faf-8cc1-440a-bba1-a95a2b05d8a5">

Current OUTPUT :- 
<img width="331" alt="image" src="https://github.com/user-attachments/assets/85a4cf78-7246-408c-a44f-53c6942d51fe">

How to apply padding/margin before starting the step and ending the step after that sync position of the Thumb in the center?

  ```
      <Slider
          style={{ marginHorizontal: 10, borderRadius: 30 }}
          progress={progress}
          minimumValue={min}
          maximumValue={max}
          step={10}
          renderMark={() => {
            return (
              <View
                style={{
                  height: 15,
                  borderRadius: 30,
                  backgroundColor: Color.GRAYSCALE,
                  borderWidth: 2,
                  borderColor: Color.GRAYSCALE,
                }}
              />
            );
          }}
          theme={{
            maximumTrackTintColor: Color.GRAYSCALE,
            minimumTrackTintColor: Color.GRAYSCALE,
            cacheTrackTintColor: Color.GRAYSCALE,
          }}
          thumbWidth={30}
          renderThumb={() => {
            return (
              <View
                style={{
                  width: 24,
                  height: 24,
                  borderRadius: 12,
                  backgroundColor: Color.VERMILION,
                }}
              />
            );
          }}
          renderBubble={() => {
            return <></>;
          }}
          snapToStep={true}
        />
```

**Comment by DuLe-FinX** [comment]
Same issue. 

@alantoa 
This issue makes the UI is very bad.
Help me fix it soon.

**Comment by alantoa** [comment]
This issue should be resolved in [v2.6.2](https://github.com/alantoa/react-native-awesome-slider/releases/tag/v2.6.2). You can check out the [latest example](https://github.com/alantoa/react-native-awesome-slider/blob/v2.6.2/example/src/sample/src/binance-slider.tsx), I think it already includes most use cases.
